### DISCUSSION TITLE
[docs]: update repo status tables for all pattern pages

### DISCUSSION THREAD

**Discussion by eyevana** [RedHat-UX/red-hat-design-system#1173] [General]
We need to update the `repoStatus.csv` file for all Pattern pages that should be rendering a repo status table.

https://github.com/RedHat-UX/red-hat-design-system/pull/1057#discussion_r1235409243

**Comment by bennypowers** [comment]
Is this still relevant?

**Comment by coreyvickery** [comment]
> Is this still relevant?

@bennypowers Yes, but perhaps it would be easier for a developer to track this information down instead of me.

Or a larger conversation is needed about how exactly we keep track of all of our elements in patterns in different repos other than `RHDS`. I believe other design system websites feature this information on individual pages, I don't know how they do it, but it would be nice to fit in some automation.

**Comment by markcaron** [comment]
Let's discuss whether we even need this table, and how we're representing Elements and Patterns and what "repos" they're in (e.g. design kits, FTS, etc.). Similar to the Status page.

And, for now, we should hide it from the Elements (and Patterns) pages.

**Comment by markcaron** [comment]
I made an issue to temporarily hide these sections: https://github.com/RedHat-UX/red-hat-design-system/issues/1174
